hw3/hw3.tex

```diff
@@ -47,12 +47,11 @@
 \end{question}
 
 Let $G$ be a graph of 6 vertices in the set $v$, named $a, b, c, d, e$, and $f$ \\ 
-If the degree of any vertex is 2, the three unconnected vertices form an independent set of size 3.  Therefore, in order to have an independent set of less than 3, every vertex needs to have a degree of at least 3.  If we build a 3-regular graph with 6 vertices, it will turn out as follows:\\
-We start by connecting $a$ to $b, c,$ and  $d$. $b, c,$ or $d$ cannot be adjacent, since that would form a clique of size 3 with $a$. In so doing, we create an independent set of size 3 with $b, c, $ and $d$.\\
-
-According to Ramsey's Theorem, the value of $R(3,3)$ is ${(3+3-2) \choose (3-1)}$, or ${4 \choose 2} = 6$, meaning a graph with a guarnateed idependent set of size 3 or a clique of size 3 must have a $V(G) \geq 6$
+If the degree of any vertex is 2, the three unconnected vertices form an independent set of size 3.  Therefore, in order to have an independent set of less than 3, a vertex needs to have a degree of at least 3.\\
 
+Let us assume that $a$ is the vertex with degree 3, and it is adjacent to $b, c, d$. If any of $b, c,$ or $d$ are adjacent to each other, they would form a clique of size 3 with $a$. If $b, c,$ or $d$ are non-adjacent they would form an independent set of size 3.\\
 
+According to Ramsey's Theorem, the value of $R(3,3)$ is ${(3+3-2) \choose (3-1)}$, or ${4 \choose 2} = 6$, meaning a graph with a guaranteed independent set of size 3 or a clique of size 3 must have a $V(G) \geq 6$
```

Comment:
I thought about removing this, since it didn't have much to do with the proof. But it's a pretty cool fact, so let's leave it in